setup.py

```diff
@@ -23,7 +23,7 @@
     "sqlparse",
     "ipython-genutils>=0.1.0",
     "jinja2",
-    "sqlglot>=11.3.7, <20.0.0",
+    "sqlglot>=11.3.7",
```

Comment:
is this the right version range? if jupysql broke in version 20.0.0, I'd expect this to be `>=20.0.0` (?)

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/966/commits/dd6c12a20a41ec362234195ae199c5dc136d0e2f)

Reply:
I wasn't suggesting that the version *must* be `>=20.0.0` but instead asking the reasoning. did you check that we actually need `>=20.0.0`?

Reply:
ok I misunderstood.
I did test with versions < 20.0.0 and I don't see any issues with the test, because now we are directly mocking the `Tokenizer` of `MySQL` class instead of relying on return value of `get_and_raise`.

<img width="894" alt="Screenshot 2023-12-21 at 10 19 44 AM" src="https://github.com/ploomber/jupysql/assets/29096135/cbb08ae0-9151-40f4-917f-c179fff5acd0">

<img width="899" alt="Screenshot 2023-12-21 at 10 20 24 AM" src="https://github.com/ploomber/jupysql/assets/29096135/3d276d45-4d6a-4b1c-8059-9f44101b0c57">

Should I revert back to `sqlglot>=11.3.7`? i'll have to revert the lock file also in that case

@edublancas

Reply:
yes, please revert

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/966/commits/233bb0ead2b783713f1f64092bcf78ec3ed1ad38)